packages/lab/src/list-next/ListNext.tsx

```diff
@@ -98,7 +112,7 @@ export const ListNext = forwardRef<HTMLUListElement, ListNextProps>(
           id={listId}
           className={clsx(withBaseName(), className)}
           role="listbox"
-          tabIndex={disabled ? -1 : 0}
+          tabIndex={disabled || controlled ? -1 : 0}
```

Comment:
I'm wondering why `controlled` would change behaviour of a component?

Reply:
changed for disableFocus